ihmc-robotics-toolkit/src/main/java/us/ihmc/robotics/MatrixMissingTools.java

```diff
@@ -465,11 +465,41 @@ public static void unsafe_add(DMatrixRMaj matrix, int row, int col, double value
    /**
     * Negates the input matrix in-place.
     *
-    * @param matrix the matrix to be negated.
+    * @param matrixToNegate the matrix to be negated. Modified.
```

Comment:
Could use CommonOps.changeSign instead

Reply:
I didn't know that was a thing, so that's pretty cool. I went through and replaced the base use of negate with that. I did keep (and rename) negate_unsafe, because that's useful when you don't want to do it in-place.